components/network/tor/tor.p5m

```diff
@@ -33,13 +35,20 @@ dir path=var/log/tor group=daemon mode=0755 owner=daemon
 file etc/tor/torrc.sample path=etc/tor/torrc mode=0644 preserve=true
 
 file usr/bin/tor path=usr/lib/tor mode=0555
```

Comment:
Is there a reason to keep the `tor` binary in `usr/lib/`? If you decide to change it, make sure `files/tor.sh` is changed accordingly.

Reply:
i used the p5m-file from the old version as a template.
In the past many programs were installed in the lib-directory :)
In my new version the tor-binary is in the (/usr/)bin-dir. I changed the tor.sh script for the new position.
At the moment i am testing this new package.

Reply:
@svschmel Cool, I'll review once you push those changes.